### PR TITLE
chapter 12

### PR DESCRIPTION
**Pull request by maihuysl1999**
Password Reset
Kiến thức cần nắm: https://docs.google.com/document/d/1L9dlWmhGjGeCMPiZD5of0TF-LqGJJK9PEh6HX9fVXcY/edit
Screenshot from 2021-08-05 21-12-26

### CODE REVIEW

**Comment by maihuysl1999** [comment]
ready

File: config/routes.rb [hunk 1]
```diff
@@ -1,4 +1,6 @@
 Rails.application.routes.draw do
+  get 'password_resets/new'
```
Comment (ngocvt-0484) [inline config/routes.rb]: thừa k e

File: config/routes.rb [hunk 2]
```diff
@@ -9,5 +11,6 @@
     delete "/logout", to: "sessions#destroy"
     resources :users
     resources :account_activations, only: :edit
+    resources :password_resets, only: [:new, :create, :edit, :update]
```
Comment (ngocvt-0484) [inline config/routes.rb]: dùng %i đi

**Comment by maihuysl1999** [comment]
ready

**Comment by ngocvt-0484** [comment]
merged

**Pull request merged**